blocklist-processing-engine/src/redis-client/redis-client.ts

```diff
@@ -0,0 +1,99 @@
+import { RedisClient } from "redis";
+import { readFile } from 'fs';
+import { configuration } from "../config/config";
+
+const client: RedisClient = new RedisClient({
+    db: configuration.redisDB,
+    host: configuration.redisHost,
+    port: configuration.redisPort,
+});
+
+/**
+ * Clears Redis, then loads sample data from file, then publishes all to Redis.
+ */
+const initialize = async () => {
+    client.flushall();
+    let sampleData: string[] = await loadData();
+    await block(sampleData);
+}
+
+/**
+ * Load all data from the file to Redis
+ */
+const loadData = (): Promise<string[]> => {
+    return new Promise(resolve => {
+        readFile('./src/data/Blocked Ghana numbers.txt', (error, data) => {
```

Comment:
We're using sample data now, but how are we going to import the given numbers in a production environment? this is probably a feature by itself.

Reply:
for POC this is what was agreed. 

for MVP / later we'd probably have a DB where this would be read from or something.